doc/getting_started/toolchain_3rd_party_x_compilers.rst

```diff
@@ -6,6 +6,25 @@
 A "3rd party toolchain" is an officially supported toolchain provided by an
 external organization. Several of these are available.
 
+.. _toolchain_ubuntu:
+
+Debian, Ubuntu
+**************
+
+#. Install a prebuilt toolchain for your operating system
+
+   .. code-block:: console
+
+      # Linux:
+      $ apt install gcc-arm-none-eabi
+
+#. Then, set the required environment variables
+
+   .. code-block:: console
+
+      $ export $ZEPHYR_TOOLCHAIN_VARIANT=cross-compile
+      $ export $CROSS_COMPILE=/usr/bin/arm-none-eabi-
+
```

Comment:
Remove, this is already documented here: https://docs.zephyrproject.org/latest/getting_started/toolchain_other_x_compilers.html#other-cross-compilers

Reply:
Sure. I was actually asked to add it in this very PR, but will do.